data/trie/trieStorageManager.go

```diff
@@ -511,17 +513,17 @@ func (tsm *trieStorageManager) removeSnapshot() {
 
 func disconnectSnapshot(db data.DBWriteCacher) error {
 	err := db.Close()
-	if err != nil {
+	/*if err != nil {
 		log.Error("trie storage manager: disconnectSnapshot", "error", err.Error())
 		return err
-	}
+	}*/
 
-	return nil
+	return err
 }
 
 func removeSnapshot(db data.DBWriteCacher, path string) {
-	err := disconnectSnapshot(db)
-	if err != nil {
+	if err := disconnectSnapshot(db); err != nil {
```

Comment:
I think it will look better if you split it like this:

err := disconnectSnapshot(s.DBWriteCacher)
if err != nil .....
 
Also, we use the same pattern throughout the rest of the project.

Reply:
Done!